### PR TITLE
Change how to release python dist

### PR DESCRIPTION
**Pull request by zhongjiajie**

### CODE REVIEW

**Comment by zhongjiajie** [comment]
Hi @Tianqi-Dotes This PR only upgrade the release process, but I found out we have markdown anchor failed, so I disable to check to all of them

File: docs/en-us/2.0.0/user_doc/guide/project/workflow-definition.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
 3. Select "Task Priority": When the number of worker threads is insufficient, high-level tasks will be executed first in the execution queue, and tasks with the same priority will be executed in the order of first in, first out;
 4. Timeout alarm (optional): Check the timeout alarm, timeout failure, and fill in the "timeout period". When the task execution time exceeds **timeout period**, an alert email will be sent and the task timeout fails;
 5. Resources (optional). Resource files are files created or uploaded on the Resource Center -> File Management page. For example, the file name is `test.sh`, and the command to call the resource in the script is `sh test.sh`;
-6. Custom parameters (optional), refer to [Custom Parameters](#UserDefinedParameters);
+6. Custom parameters (optional);
```
Comment (Tianqi-Dotes) [inline docs/en-us/2.0.0/user_doc/guide/project/workflow-definition.md:20]: A verb + noun.
Maybe 
Customize parameters (optional);
is better

File: docs/en-us/2.0.1/user_doc/guide/project/workflow-definition.md [hunk 2]
```diff
@@ -17,7 +17,7 @@
 3. Select "Task Priority": When the number of worker threads is insufficient, high-level tasks will be executed first in the execution queue, and tasks with the same priority will be executed in the order of first in, first out;
 4. Timeout alarm (optional): Check the timeout alarm, timeout failure, and fill in the "timeout period". When the task execution time exceeds **timeout period**, an alert email will be sent and the task timeout fails;
 5. Resources (optional). Resource files are files created or uploaded on the Resource Center -> File Management page. For example, the file name is `test.sh`, and the command to call the resource in the script is `sh test.sh`;
-6. Custom parameters (optional), refer to [Custom Parameters](#UserDefinedParameters);
+6. Custom parameters (optional);
```
Comment (Tianqi-Dotes) [inline docs/en-us/2.0.1/user_doc/guide/project/workflow-definition.md:20]: same as above

File: docs/en-us/2.0.3/user_doc/guide/project/workflow-definition.md [hunk 3]
```diff
@@ -17,7 +17,7 @@
 3. Select "Task Priority": When the number of worker threads is insufficient, high-level tasks will be executed first in the execution queue, and tasks with the same priority will be executed in the order of first in, first out;
 4. Timeout alarm (optional): Check the timeout alarm, timeout failure, and fill in the "timeout period". When the task execution time exceeds **timeout period**, an alert email will be sent and the task timeout fails;
 5. Resources (optional). Resource files are files created or uploaded on the Resource Center -> File Management page. For example, the file name is `test.sh`, and the command to call the resource in the script is `sh test.sh`;
-6. Custom parameters (optional), refer to [Custom Parameters](#UserDefinedParameters);
+6. Custom parameters (optional);
```
Comment (Tianqi-Dotes) [inline docs/en-us/2.0.3/user_doc/guide/project/workflow-definition.md:20]: same as above

File: docs/en-us/2.0.5/user_doc/guide/project/workflow-definition.md [hunk 4]
```diff
@@ -17,7 +17,7 @@
 3. Select "Task Priority": When the number of worker threads is insufficient, high-level tasks will be executed first in the execution queue, and tasks with the same priority will be executed in the order of first in, first out;
 4. Timeout alarm (optional): Check the timeout alarm, timeout failure, and fill in the "timeout period". When the task execution time exceeds **timeout period**, an alert email will be sent and the task timeout fails;
 5. Resources (optional). Resource files are files created or uploaded on the Resource Center -> File Management page. For example, the file name is `test.sh`, and the command to call the resource in the script is `sh test.sh`;
-6. Custom parameters (optional), refer to [Custom Parameters](#UserDefinedParameters);
+6. Custom parameters (optional);
```
Comment (Tianqi-Dotes) [inline docs/en-us/2.0.5/user_doc/guide/project/workflow-definition.md:20]: same as above

File: docs/en-us/dev/user_doc/guide/project/workflow-definition.md [hunk 5]
```diff
@@ -17,7 +17,7 @@
 3. Select "Task Priority": when the number of worker threads is insufficient, high priority tasks will execute first in the execution queue, and tasks with the same priority will execute in the order of first in, first out;
 4. Timeout alarm (optional): check the timeout alarm, timeout failure, and fill in the "timeout period". When the task execution time exceeds **timeout period**, an alert email will send and the task timeout fails;
 5. Resources (optional). Resources are files create or upload in the Resource Center -> File Management page. For example, the file name is `test.sh`, and the command to call the resource in the script is `sh test.sh`;
-6. Custom parameters (optional), refer to [Custom Parameters](#UserDefinedParameters);
+6. Custom parameters (optional);
```
Comment (Tianqi-Dotes) [inline docs/en-us/dev/user_doc/guide/project/workflow-definition.md:20]: same as above

File: docs/en-us/2.0.0/user_doc/guide/project/workflow-definition.md [hunk 6]
```diff
@@ -49,7 +49,7 @@ The operation functions of the workflow definition list are as follows:
 - **Edit:** Only "offline" workflow definitions can be edited. Workflow DAG editing is the same as [Create Workflow Definition](#create-workflow-definition).
 - **Online:** When the workflow status is "Offline", used to online workflow. Only the workflow in the "Online" state can run, but cannot be edited.
 - **Offline:** When the workflow status is "Online", used to offline workflow. Only the workflow in the "Offline" state can be edited, but not run.
-- **Run:** Only workflow in the online state can run. See [2.3.3 Run Workflow](#runWorkflow) for the operation steps
+- **Run:** Only workflow in the online state can run. See [2.3.3 Run Workflow](#run-the-workflow) for the operation steps
```
Comment (Tianqi-Dotes) [inline docs/en-us/2.0.0/user_doc/guide/project/workflow-definition.md:52]: forget '.' at the end of the sentence.

File: docs/en-us/2.0.1/user_doc/guide/project/workflow-definition.md [hunk 7]
```diff
@@ -49,7 +49,7 @@ The operation functions of the workflow definition list are as follows:
 - **Edit:** Only "offline" workflow definitions can be edited. Workflow DAG editing is the same as [Create Workflow Definition](#create-workflow-definition).
 - **Online:** When the workflow status is "Offline", used to online workflow. Only the workflow in the "Online" state can run, but cannot be edited.
 - **Offline:** When the workflow status is "Online", used to offline workflow. Only the workflow in the "Offline" state can be edited, but not run.
-- **Run:** Only workflow in the online state can run. See [2.3.3 Run Workflow](#runWorkflow) for the operation steps
+- **Run:** Only workflow in the online state can run. See [2.3.3 Run Workflow](#run-the-workflow) for the operation steps
```
Comment (Tianqi-Dotes) [inline docs/en-us/2.0.1/user_doc/guide/project/workflow-definition.md:52]: same as above

File: docs/en-us/dev/user_doc/guide/project/workflow-definition.md [hunk 8]
```diff
@@ -49,7 +49,7 @@ The following are the operation functions of the workflow definition list:
 - **Edit:** Only "Offline" workflow definitions can be edited. Workflow DAG editing is the same as [Create Workflow Definition](#creatDag)
 - **Online:** When the workflow status is "Offline", used to make workflow online. Only the workflow in the "Online" state can run, but cannot edit
 - **Offline:** When the workflow status is "Online", used to make workflow offline. Only the workflow in the "Offline" state can be edited, but cannot run
-- **Run:** Only workflow in the online state can run. See [2.3.3 Run Workflow](#runWorkflow) for the operation steps
+- **Run:** Only workflow in the online state can run. See [2.3.3 Run Workflow](#run-the-workflow) for the operation steps
```
Comment (Tianqi-Dotes) [inline docs/en-us/dev/user_doc/guide/project/workflow-definition.md:52]: same as above

File: docs/zh-cn/2.0.5/user_doc/guide/project/workflow-definition.md [hunk 9]
```diff
@@ -47,7 +47,7 @@
 - **编辑：** 只能编辑"下线"的工作流定义。工作流DAG编辑同创建工作流定义。
 - **上线：** 工作流状态为"下线"时，上线工作流，只有"上线"状态的工作流能运行，但不能编辑。
 - **下线：** 工作流状态为"上线"时，下线工作流，下线状态的工作流可以编辑，但不能运行。
-- **运行：** 只有上线的工作流能运行。运行操作步骤见[2.3.3 运行工作流](#runWorkflow)
+- **运行：** 只有上线的工作流能运行。运行操作步骤见运行工作流
```
Comment (Tianqi-Dotes) [inline docs/zh-cn/2.0.5/user_doc/guide/project/workflow-definition.md:50]: same as above

File: docs/en-us/2.0.3/user_doc/guide/project/workflow-definition.md [hunk 10]
```diff
@@ -102,7 +102,7 @@ The operation functions of the workflow definition list are as follows:
   <p align="center">
       <img src="/img/timer-en.png" width="80%" />
   </p>
-- Failure strategy, notification strategy, process priority, worker group, notification group, recipient, and CC are the same as [workflow running parameters](#runParamers).
+- Failure strategy, notification strategy, process priority, worker group, notification group, recipient, and CC are the same as workflow running parameters
```
Comment (Tianqi-Dotes) [inline docs/en-us/2.0.3/user_doc/guide/project/workflow-definition.md:105]: same as above

File: docs/en-us/dev/user_doc/guide/project/workflow-definition.md [hunk 11]
```diff
@@ -102,7 +102,7 @@ The following are the operation functions of the workflow definition list:
   <p align="center">
       <img src="/img/timer-en.png" width="80%" />
   </p>
-- Failure strategy, notification strategy, process priority, worker group, notification group, recipient, and CC are the same as [workflow running parameters](#runParamers).
+- Failure strategy, notification strategy, process priority, worker group, notification group, recipient, and CC are the same as workflow running parameters
```
Comment (Tianqi-Dotes) [inline docs/en-us/dev/user_doc/guide/project/workflow-definition.md:105]: same as above

File: docs/zh-cn/2.0.0/user_doc/guide/project/workflow-definition.md [hunk 12]
```diff
@@ -99,7 +99,7 @@
     <p align="center">
         <img src="/img/time-schedule2.png" width="80%" />
     </p>
-- 失败策略、通知策略、流程优先级、Worker分组、通知组、收件人、抄送人同[工作流运行参数](#runParamers)。
+- 失败策略、通知策略、流程优先级、Worker分组、通知组、收件人、抄送人同工作流运行参数
```
Comment (Tianqi-Dotes) [inline docs/zh-cn/2.0.0/user_doc/guide/project/workflow-definition.md:102]: same as above

File: docs/zh-cn/2.0.2/user_doc/guide/project/workflow-definition.md [hunk 13]
```diff
@@ -99,7 +99,7 @@
     <p align="center">
         <img src="/img/time-schedule2.png" width="80%" />
     </p>
-- 失败策略、通知策略、流程优先级、Worker分组、通知组、收件人、抄送人同[工作流运行参数](#runParamers)。
+- 失败策略、通知策略、流程优先级、Worker分组、通知组、收件人、抄送人同工作流运行参数
```
Comment (Tianqi-Dotes) [inline docs/zh-cn/2.0.2/user_doc/guide/project/workflow-definition.md:102]: same as above

File: docs/zh-cn/2.0.3/user_doc/guide/project/workflow-definition.md [hunk 14]
```diff
@@ -99,7 +99,7 @@
     <p align="center">
         <img src="/img/time-schedule2.png" width="80%" />
     </p>
-- 失败策略、通知策略、流程优先级、Worker分组、通知组、收件人、抄送人同[工作流运行参数](#runParamers)。
+- 失败策略、通知策略、流程优先级、Worker分组、通知组、收件人、抄送人同工作流运行参数
```
Comment (Tianqi-Dotes) [inline docs/zh-cn/2.0.3/user_doc/guide/project/workflow-definition.md:102]: same as above

File: docs/en-us/2.0.0/user_doc/guide/project/workflow-instance.md [hunk 15]
```diff
@@ -17,7 +17,7 @@
    <p align="center">
      <img src="/img/instanceViewLog-en.png" width="80%" />
    </p>
-- Click "View Log", a log pop-up box will pop up, as shown in the figure below, the task log can also be viewed on the task instance page, refer to [Task View Log](#taskLog)。
+- Click "View Log", a log pop-up box will pop up, as shown in the figure below, the task log can also be viewed on the task instance page, refer to [Task View Log](./task-instance.md)。
```
Comment (Tianqi-Dotes) [inline docs/en-us/2.0.0/user_doc/guide/project/workflow-instance.md:20]: change '。' to '.'

File: docs/en-us/2.0.1/user_doc/guide/project/workflow-instance.md [hunk 16]
```diff
@@ -17,7 +17,7 @@
    <p align="center">
      <img src="/img/instanceViewLog-en.png" width="80%" />
    </p>
-- Click "View Log", a log pop-up box will pop up, as shown in the figure below, the task log can also be viewed on the task instance page, refer to [Task View Log](#taskLog)。
+- Click "View Log", a log pop-up box will pop up, as shown in the figure below, the task log can also be viewed on the task instance page, refer to [Task View Log](./task-instance.md)。
```
Comment (Tianqi-Dotes) [inline docs/en-us/2.0.1/user_doc/guide/project/workflow-instance.md:20]: same as above

File: docs/en-us/2.0.2/user_doc/guide/project/workflow-instance.md [hunk 17]
```diff
@@ -17,7 +17,7 @@
    <p align="center">
      <img src="/img/instanceViewLog-en.png" width="80%" />
    </p>
-- Click "View Log", a log pop-up box will pop up, as shown in the figure below, the task log can also be viewed on the task instance page, refer to [Task View Log](#taskLog)。
+- Click "View Log", a log pop-up box will pop up, as shown in the figure below, the task log can also be viewed on the task instance page, refer to [Task View Log](./task-instance.md)。
```
Comment (Tianqi-Dotes) [inline docs/en-us/2.0.2/user_doc/guide/project/workflow-instance.md:20]: same as above

File: docs/en-us/2.0.5/user_doc/guide/project/workflow-instance.md [hunk 18]
```diff
@@ -17,7 +17,7 @@
    <p align="center">
      <img src="/img/instanceViewLog-en.png" width="80%" />
    </p>
-- Click "View Log", a log pop-up box will pop up, as shown in the figure below, the task log can also be viewed on the task instance page, refer to [Task View Log](#taskLog)。
+- Click "View Log", a log pop-up box will pop up, as shown in the figure below, the task log can also be viewed on the task instance page, refer to [Task View Log](./task-instance.md)。
```
Comment (Tianqi-Dotes) [inline docs/en-us/2.0.5/user_doc/guide/project/workflow-instance.md:20]: same as above

File: docs/en-us/2.0.2/user_doc/guide/project/workflow-definition.md [hunk 19]
```diff
@@ -1,6 +1,6 @@
 # Workflow definition
 
-## <span id=creatDag> Create workflow definition</span>
+## Create workflow definitionc
```
Comment (Tianqi-Dotes) [inline docs/en-us/2.0.2/user_doc/guide/project/workflow-definition.md]: wrong word  spelling 'definitionc'

**Review by Tianqi-Dotes — COMMENTED** [review]
Dear Jiajie,
Please check the comments.
Yours 
Tianqi

**Review by Tianqi-Dotes — APPROVED** [review]
LGTM

**Pull request merged**